compiler/rustc_mir/src/transform/mod.rs

```diff
@@ -625,3 +628,27 @@ fn promoted_mir<'tcx>(
 
     tcx.arena.alloc(promoted)
 }
+
+pub(crate) fn custom_coerce_unsize_info<'tcx>(
```

Comment:
Maybe this function would be better put somewhere in `rustc_mir/src/util`?

Reply:
should i create a new module in `util` for this?

Reply:
If none of the others seem to fit, yeah.